### PR TITLE
Make card frame scroll before card content

### PR DESCRIPTION
**Pull request by katevoss**
Closes https://github.com/catalpainternational/asteroid/issues/44

### CODE REVIEW

**Comment by sandroroux** [comment]
@katevoss, `LessonTest` lacks `CardFrame`, but `LessonTest`'s bottom glitches on small, scrollable screens. Should we fix the glitch separately?

image

**Comment by sandroroux** [comment]
Note that the feedback card overflows narrow screens. The inner content should respond to the outer limits. Perhaps small screens should stack the buttons vertically (flexbox style):
image

File: src/scss/_lesson.scss [hunk 1]
```diff
@@ -217,12 +217,23 @@ LessonPage .description {
...
 }
 
+LessonComplete, LessonModuleComplete, LessonFeedback {
+    LessonFrame {
+        .frame-background {
+            min-height: 550px;
+        }
+
+        .card-frame {
+            min-height: 460px;
```
Comment (sandroroux) [inline src/scss/_lesson.scss:227]: Both heights should be `auto`. I may misunderstand the requirements, but the card should stretch to the contents.

File: src/scss/_lesson.scss [hunk 2]
```diff
@@ -217,12 +217,23 @@ LessonPage .description {
     }
 }
 
+LessonComplete, LessonModuleComplete, LessonFeedback {
```
Comment (sandroroux) [inline src/scss/_lesson.scss:220]: I'm also curious these styles only apply to these three views. I'm hoping `LessonFrame` can tolerate any use case, plug-and-play.

**Comment by PeteCoward** [comment]
Leaving this open pending Kate's comments on Alessandro's comments

**Comment by sandroroux** [comment]
@katevoss, I'm bumping this in case you missed @PeteCoward's comment.

**Comment by katevoss** [comment]
@sandroroux would you mind taking a look at my change here https://github.com/catalpainternational/canoe/pull/47/commits/013407a708aea407fa6da38ffe349b98485e8fcb ? I'm having some trouble getting the Feedback card content to fill to the size of the contents with `auto`:
image

**Comment by sandroroux** [comment]
@katevoss I can't reproduce that shrunken card. How did you get it to appear?

Also, make sure that in `_lesson` L214-223, the `min-height`s become `height`s:
```sass
LessonComplete, LessonModuleComplete, LessonFeedback {
    LessonFrame {
        .frame-background {
            height: auto;
        }

        .card-frame {
            height: auto;
        }
    }
}
```

**Comment by PeteCoward** [comment]
merging to tidy up but will leave #60 open for improvements

**Pull request merged**